wtpython/__main__.py

```diff
@@ -29,34 +31,38 @@ def parse_arguments() -> tuple[dict, list]:
     )
 
     flags, args = parser.parse_known_args()
+
+    # Set sys.argv as the intended script would receive them
+    sys.argv = ['python', *args[1:]]
     return vars(flags), args
 
 
 def main() -> None:
     """Run the application"""
     flags, args = parse_arguments()
-    tb = run_and_get_stderr(args)
-    if not tb:
-        return
+    exc = None
+
```

Comment:
Perhaps here would be a nice place to replace `sys.argv`, and restore it after (before returning the exception)

Reply:
recreated a function for running that will alter `sys.argv` then reset in a `finally` block